packages/turbo-ignore/package.json

```diff
@@ -25,16 +25,18 @@
     "check-types": "tsc --noEmit",
     "lint:prettier": "prettier -c . --cache --ignore-path=../../.prettierignore"
   },
+  "dependencies": {
+    "commander": "^11.0.0",
```

Comment:
This is an old trick to improve the `npx` install time because dev dependencies get included with this packages bundle, so there's no extra fetch to install dependencies.

We did this explicitly for `turbo-ignore` to ensure it installs fast as possible when being npx'd in ci systems.